tree/dataframe/src/RDataFrame.cxx

```diff
@@ -1232,26 +1232,56 @@ Notice how we created one `double` variable for each processing slot and later m
 
 
 \anchor friends
-### Friend trees
-Friend TTrees are supported by RDataFrame.
-Friend TTrees with a TTreeIndex are supported starting from ROOT v6.24.
+### Dataset joins with friend trees
 
-To use friend trees in RDataFrame, it is necessary to add the friends directly to
-the tree and instantiate an RDataFrame with the main tree:
+Vertically concatenating multiple trees that have the same columns (creating a logical dataset with the same columns and
+more rows) is trivial in RDataFrame: just pass all tree and file names to RDataFrame's constructor, or create a TChain
```

Comment:
> just pass all tree and file names to RDataFrame's constructor

There is no constructor that accepts more than one tree at the same time, except indirectly through `TChain` or `RDatasetSpec`. Maybe it could be changed to 

```suggestion
more rows) is trivial in RDataFrame: just pass the tree name and a list of file names to RDataFrame's constructor, or create a TChain
```

Reply:
good point, thanks!